jobbergateapi2/main.py

```diff
@@ -0,0 +1,34 @@
+"""
+Main file to startup the fastapi server
+"""
+from fastapi import FastAPI
+from gino.ext.starlette import Gino
+from mangum import Mangum
+
+from .config import settings
```

Comment:
Stylistically, please use absolute imports throughout, e.g. `from jobbergateapi2.config ..`

Relative imports can lead to surprises when you refactor.